src/dsp_tools/excel2xml.py

```diff
@@ -1217,7 +1217,19 @@ def make_text_prop(
             **kwargs,
             nsmap=xml_namespace_map
         )
-        value_.text = val.value
+        if kwargs["encoding"] == "utf8":
+            # write the text into the tag, without validation
+            value_.text = str(val.value)
+        else:
+            # enforce that the text is well-formed XML: serialize tag, insert text, and parse it again
+            value_.text = "replace-me_8(1-)replace-me"
+            content = etree.tostring(value_, encoding="unicode")
+            content = content.replace("replace-me_8(1-)replace-me", str(val.value))
+            try:
+                value_ = etree.fromstring(content)
+            except etree.XMLSyntaxError:
+                raise BaseError("The XML tags contained in a richtext property (encoding=xml) must be well-formed. "
+                                "The special characters <, > and & are only allowed to construct a tag.") from None
```

Comment:
just curious: what does `from None` do here?

Reply:
That's an exciting new thing that I just learned. In fact, there are many cases in the DSP-TOOLS codebase where I try to beautify an error with 

```
try:
    something()
except DifficultErrorWithComplicatedStacktrace:
    raise BaseError("Nice Message with simpler stacktrace")
```

Now I just learnt that this doesn't work as intended. In fact, the result is even uglier than what I tried to prevent: In addition to the complicated stacktrace, I get the following: `During handling of the above exception, another exception occurred`.

The solution is to add `from None` behind the `raise` statement. It is explained here: https://stackoverflow.com/questions/52725278/#comment103220039_52725410.

In a separate PR, I will add `from None` to all occurrences in the codebase.

Reply:
I see, nice